lblod/pipeline.py

```diff
@@ -0,0 +1,90 @@
+import json
+import datetime
+import os
+import uuid
+import gzip
+from rdflib import Graph
+
+import requests
+from constants import DEFAULT_GRAPH, TASK_STATUSES
+
+from lblod.file import STORAGE_PATH, construct_insert_file_query
+from lblod.harvester import collection_has_collected_files, create_results_container
+from lblod.job import update_task_status
+from sudo_query import update_sudo
+from helpers import logger
+
+
+def get_item(rdo, task):
+    # Get all possible urls
+    json_urls = {"urls": ["https://verenigingen.oscart-dev.s.redhost.be/json-ld?page=1",
```

Comment:
I'm a bit confused here. those url should come from the task created by either an user or a cron scheduler, it seems they are hardcoded.
You don't seem to use rdo and task in that function

Reply:
Yes, so for the moment we do not have access to the API that we should harvest but from what the provider told us, with url that the user will input, we will get a response that looks like this 
```
{[{vcode: V100100}, {vcode: V100101}]}
```
but we want to scrape the detail page of that API, so URL/detail/V100100 and URL/detail/V100101 . As we don't have the API yet, we just mocked this.